### DISCUSSION TITLE
my slicer parameters

### DISCUSSION THREAD

**Discussion by sobhonium** [sn4k3/UVtools#713] [General]
There are some parameters of SLA printing (which are visible in Anycube Workshop slicer tool) that I cannot find in prusa folder in uvtools or somewhere else. They include 'time-off', 'retracting', etc. How can I add them to my final exported file. I've checked .ini files and see  that it might probably be OK-ish to add those parameters in a format like time-off=5 in .ini files. Now, I don't know what I should name these parameters in .ini files. Can you please help me out?

**Comment by sn4k3** [comment]
To configure those on PrusaSlicer you need to access printer tab and go to notes. Write values using that notes: https://github.com/sn4k3/UVtools#prusaslicer

time-off = Wait time before cure = WaitTimeBeforeCure_xxx
Retracting = Retract speed = RetractSpeed_xxx

A time off of 5s would be set as `WaitTimeBeforeCure_5`

**Reply by sobhonium** [reply]
I use two files one is prusuaslicer.ini  like  bellow
//////////////////////////////////////////////
[DEFAULT]
absolute_correction = 0
area_fill = 50
bed_custom_model = 
bed_custom_texture = 
bed_shape = 0x0,198.15x0,198.15x123.84,0x123.84
bottle_cost = 0
bottle_volume = 1000
bottle_weight = 1
compatible_printers_condition_cummulative = "printer_model==\"SL1\"";"printer_model==\"SL1\""
compatible_prints_condition = 0.05
layer_height = 0.05
compatible_prints_condition_cummulative = "layer_height == 0.05"
default_sla_material_profile = Prusa Orange Tough 0.05
default_sla_print_profile = 0.05 Normal
display_height = 123.84
display_mirror_x = 1
display_mirror_y = 0
display_orientation = landscape
display_pixels_x = 5760
display_pixels_y = 3600
display_width = 198.15
elefant_foot_compensation = 0.2
elefant_foot_min_width = 0.2
exposure_time = 15.0
faded_layers = 10
fast_tilt_time = 5
gamma_correction = 1
hollowing_closing_distance = 2
hollowing_enable = 0
hollowing_min_thickness = 3
hollowing_quality = 0.5
host_type = octoprint
inherits_cummulative = ;;"Original Prusa SL1"
initial_exposure_time = 40.0
initial_layer_height = 0.3
layer_height = 0.05
material_correction = 1,1,1
material_density = 1
material_notes = 
material_type = Tough
material_vendor = (Unknown)
max_exposure_time = 120
max_initial_exposure_time = 300
max_print_height = 245
min_exposure_time = 1
min_initial_exposure_time = 1
output_filename_format = [input_filename_base].sl1
pad_around_object = 0
pad_around_object_everywhere = 0
pad_brim_size = 1.6
pad_enable = 1
pad_max_merge_distance = 50
pad_object_connector_penetration = 0.3
pad_object_connector_stride = 10
pad_object_connector_width = 0.5
pad_object_gap = 1
pad_wall_height = 0
pad_wall_slope = 90
pad_wall_thickness = 1
physical_printer_settings_id = 
printer_model = SL1
printer_notes = Don't remove the following keywords! These keywords are used in the "compatible printer" condition of the print and filament profiles to link the particular print and filament profiles to this printer profile.
PRINTER_VENDOR_PRUSA3D
PRINTER_MODEL_SL1
PRINTER_VENDOR_ANYCUBIC
PRINTER_MODEL_PHOTON_MONO_X_6K
FILEVERSION_516
FILEFORMAT_PWMB

START_CUSTOM_VALUES
WaitTimeBeforeCure_2.5
BottomLiftHeight_8
LiftHeight_8
BottomLiftSpeed_120
LiftSpeed_120
RetractSpeed_120
BottomLightPWM_255
LightPWM_255
BottomLightOffDelay_0.5
LightOffDelay_0.5
END_CUSTOM_VALUES
printer_settings_id = AnyCubic Photon Mono X 6K
printer_technology = SLA
printer_variant = default
printer_vendor = AnyCubic
relative_correction = 1,1
sla_material_settings_id = Prusa Orange Tough @0.05
sla_print_settings_id = 0.05 Normal
slice_closing_radius = 0.005
slow_tilt_time = 8
support_base_diameter = 3
support_base_height = 1
support_base_safety_distance = 1
support_buildplate_only = 0
support_critical_angle = 45
support_head_front_diameter = 0.4
support_head_penetration = 0.4
support_head_width = 3
support_max_bridge_length = 10
support_max_bridges_on_pillar = 3
support_max_pillar_link_distance = 10
support_object_elevation = 5
support_pillar_connection_mode = zigzag
support_pillar_diameter = 1
support_pillar_widening_factor = 0
support_points_density_relative = 100
support_points_minimal_distance = 1
support_small_pillar_diameter_percent = 60%
supports_enable = 1
//////////////////////////////////////////////
and config.ini as bellow:

[DEFAULT]
action = print
expTime = 2.57
expTimeFirst = 13.0
expUserProfile = 0
fileCreationTimestamp = 2023-10-29 17:04:33 UTC
hollow = 0
layerHeight = 0.05
materialName = Prusa Orange Tough 0.05
numFade = 13
numFast = 401
numSlow = 0
printProfile = 0.05 Normal
printTime = 8452.0
printerModel = SL1
printerProfile = AnyCubic Photon Mono X 6K
printerVariant = default
usedMaterial = 56.17230496471621
//////////////////////////////////////////////

Then I make a .sl1s file with these files (these two .ini files plus a thumbnail folder) and convert it to .pwmb via UVtools.
The problem is with these variable like Off time(s) and z Lift Distance which are no changed when I open the .pwmb file on Anycube Workshop. Others like 'exposure time', 'bottom layers' are OK.

**Reply by sn4k3** [reply]
z Lift Distance is defined by `LiftHeight_xxx` which you have set to 8s.
You can change settings on UVtools and see on anycubic if they changed.
